### ISSUE TITLE
tslint.exclude setting not working

### ISSUE THREAD

**Issue by greglarson** [Microsoft/vscode-tslint#179]
I have set tslint.exclude to ignore spec files:
"tslint.exclude": "src/**/*.spec.ts"

This has no effect. Tslint is still reporting violations in the editor and in the Problems pane.

For instance:
TS search.component.spec.ts src/app/search
  [tslint] expected variable-declaration: 'testCity' to have a typedef (typedef) (75, 25)

**Comment by egamma** [comment]
There is an issue with relative glob patterns

```json
"tslint.exclude": "**/**/*.spec.ts"
```
will work.

**Comment by devuxer** [comment]
I must be doing something wrong, but I'm getting a series of warning for the react npm module even though I have the following in my VSCode user settings:

    "tslint.jsEnable": false,
    "tslint.autoFixOnSave": true,
    "tslint.exclude": "**/node_modules/**"

Here is the first:

> WARNING in ./~/React/react.js
> There are multiple modules with names that only differ in casing.
> This can lead to unexpected behavior when compiling on a filesystem with other case-semantic.
> Use equal casing. Compare these module identifiers:
> * [path redacted]\node_modules\React\react.js
>     Used by 1 module(s), i. e.
>     [path redacted]\node_modules\awesome-typescript-loader\dist\entry.js![path redacted]\src\components\selected-nation-list\SelectedNationListItem.tsx
> * [path redacted]\node_modules\react\react.js
>     Used by 149 module(s), i. e.
>     [path redacted]\node_modules\awesome-typescript-loader\dist\entry.js![path redacted]\src\index.tsx

There are 24 more like this.

What can I do to suppress these warning?

**Comment by egamma** [comment]
@devuxer please file a separate issue, the issue mentioned in this issue has been addressed, so this looks like another problem.

**Comment by devuxer** [comment]
That was fast...will do.

**Comment by devuxer** [comment]
Done see https://github.com/Microsoft/vscode-tslint/issues/200.
